Packs/MicrosoftGraphMail/ReleaseNotes/1_0_23.md

```diff
@@ -0,0 +1,4 @@
+
+#### Mappers
+##### New: Microsoft Graph Mail Mapper
+- Incoming mapper for phishing emails.
```

Comment:
@tomer-pan Is this mapper just for phishing and not a general mapper for Graph?

Reply:
@David-BMS Yes, as all of the other mail mappers are. It is mapping the Phishing fields since they are referring to Email fields, no other fields are relevant as far as I can see (since this mapper is only for msgraph MAIL).